src/Resources/views/layout.html.twig

```diff
@@ -341,6 +342,12 @@
                                     </section>
                                 {% endblock content_header_wrapper %}
 
+                                <div id="update_button" class="text-center invisible">This page is outdated 
```

Comment:
Couldn't we update the DOM with the new data instead of displaying this message. Maybe could we leverage Symfony UX Turbo or Symfony Live Components to do that? IMHO this is worth discussing with Javier first by opening an issue.